tests/unit/utils/test_yamldumper.py

```diff
@@ -0,0 +1,111 @@
+"""
```

Comment:
Please create this test in the `tests/pytests/unit/utils` directory. We are slowly migrating out of this paradigm